Packs/AWS-EC2/ReleaseNotes/1_2_41.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### AWS - EC2
+
+- You can now configure the integration with a list of AWS accounts (as the parameter `AWS organization accounts`) and a role name which grants full access to each of them (as the parameter `Role name for cross-organization account access`). This will allow the command `aws-ec2-describe-addresses` to run in all of the accounts specified.
```

Comment:
```suggestion
You can now configure the integration with a list of AWS accounts (as the parameter `AWS organization accounts`) and a role name which grants full access to each of them (as the parameter `Role name for cross-organization account access`). This will allow the command `aws-ec2-describe-addresses` to run in all of the accounts specified.
```